beets/autotag/hooks.py

```diff
@@ -39,8 +39,27 @@
 
 
 # Classes used to represent candidate options.
+class Map(dict):
+    """
+    Example:
+    m = Map({'first_name': 'Eduardo'}, last_name='Pool', age=24,
+    sports=['Soccer'])
```

Comment:
This class could use a more direct docstring. Also, maybe something like `AttrDict` is a better name than `Map`? A dict is already a map, after all…